src/test/java/uk/gov/companieshouse/api/strikeoffobjections/controller/ObjectionControllerTest.java

```diff
@@ -127,13 +150,18 @@ void createObjectionEligibilityErrorTest() {
 
     @Test
     void createObjectionExceptionTest() {
-        when(objectionService.createObjection(REQUEST_ID, COMPANY_NUMBER, AUTH_ID, AUTH_USER))
+        ObjectionCreate objectionCreate = new ObjectionCreate();
+        objectionCreate.setFullName(FULL_NAME);
+        objectionCreate.setShareIdentity(false);
+        when(objectionService.createObjection(eq(REQUEST_ID), eq(COMPANY_NUMBER),
+                eq(AUTH_ID), eq(AUTH_USER), any()))
```

Comment:
same here, you can replace any() with the expected objectionCreate

Reply:
Updated